src/main/scala/com/kjetland/jackson/jsonSchema/JsonSchemaGenerator.scala

```diff
@@ -1088,6 +1088,10 @@ class JsonSchemaGenerator
                   .flatMap(p => Option(p.getAnnotation(classOf[JsonSchemaProperty])))
                   .map(_.required())
 
+                val jsonIncludeOptional:Boolean = prop
```

Comment:
Can we make this match with the other booleans being evaluated for `requiredProperty`? This is looking for the negative (read: _not required_), whereas the other booleans - `jsonSchemaRequired`, `validationAnnotationRequired` - are looking for the positive (read: _is required_).

Reply:
I played with the logic of this for a bit, and inverting the variable to be required changed the semantics. The JsonInclude is being used as a signal for optional properties - the response is serialized the same as Optional<Type>.

Reply:
Sounds good, thanks for looking.